pkgs/development/node-packages/node-packages.json

```diff
@@ -289,6 +289,46 @@
 , "textlint-rule-unexpanded-acronym"
 , "textlint-rule-write-good"
 , "thelounge"
+, "thelouge-theme-infinite"
```

Comment:
Might remove this one. It doesn't have a source repository attached to it's NPM entry, and it hasn't been updated in 2 years. (I automated the retrieval of this list, so that's my bad.)